### DISCUSSION TITLE
Can't login with Student Accounts

### DISCUSSION THREAD

**Discussion by LisaCharlotteKane** [community/community#130536] [GitHub Education]
### Hi everyone,

Hi Team. 

Our collegues at Microsoft have reached out for assistance as they shared the Github student developer pack and they are struggling to login with their student account. Domain is "rok.catholic.edu.au"

Are you able to help please

**Comment by swilson15** [comment]
Hi, @LisaCharlotteKane !

Happy to help! Can I get a little clarification on the what you mean by "struggling to login with their student account?" Do you mean they are attempting to get the student developer pack and are getting rejected because of their domain? Or, do you mean that they are having trouble verifying the domain on the GitHub account?

It would also be helpful for me attempting to resolve the issue to know which school(s) the users are attempting to apply to.
